packages/components/src/autocomplete/style.native.scss

```diff
@@ -0,0 +1,61 @@
+.container {
+	height: $mobile-header-toolbar-height;
+	background-color: $app-background;
+	border-top-color: $gray-lighten-30;
+	border-top-width: 1px;
+}
+
+.containerDark {
+	background-color: $app-background-dark-alt;
+	border-top-color: $dark-quaternary;
+	border-bottom-width: 1px;
+	border-bottom-color: $dark-quaternary;
+}
+
+.content {
+	flex-grow: 1;
+	padding-left: 6px;
+}
+
+.item {
+	flex-direction: row;
+	align-items: center;
+	padding-top: 5px;
```

Comment:
Could this be rewritten as `padding: 5px 12px 8px 8px;` ?

Reply:
Of course, thanks for the suggestion!